### ISSUE TITLE
A11y: ARIA labels and roles should be defined

### ISSUE THREAD

**Issue by beyackle** [microsoft/BotFramework-Composer#6464]
## Describe the bug

We have a large number of elements with incorrect ARIA roles, missing ARIA labels, and other related issues. This is a general problem throughout the app, and the Accessibility Insights extension is good at pointing these places out.

## Version

1.3.1
